coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -22,23 +22,21 @@
         <template slot="label">
           <div class="certify-stmt">
             I, <b>{{trimmedCertifiedBy || '[Legal Name]'}}</b>, certify that I have relevant knowledge of the
-            association and that I am authorized to make this filing.
+            {{ entityDisplay || 'association' }} and that I am authorized to make this filing.
           </div>
         </template>
       </v-checkbox>
       <p class="certify-clause">{{currentDate}}</p>
       <p class="certify-clause">
-        Note: It is an offence to make a false or misleading statement in
-        respect of a material fact in a record submitted to the Corporate Registry for filing.
-        See section 200 of the Cooperatives Association Act.
+        {{message}}
```

Comment:
I like it. Not 100% sure but linters might want spaces in between the brackets and the template string.

Reply:
Lint doesn't seem to complain on my system, but definitely let me know otherwise!